### ISSUE TITLE
style is not right for pinned column when using aligned grid as footer with ag-theme-material theme 

### ISSUE THREAD

**Issue by Teamop** [ag-grid/ag-grid#2021]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

Now, I want to use aligned grid as footer with ag-theme-material, also with **autoHeight** for the footer grid, then the header style of pinned columns is not totally right, the border bottom is not hidden.
![image](https://user-images.githubusercontent.com/8519685/32213012-6bab8c2e-be54-11e7-93b8-1f17922b68cf.png)

**Expected behavior**

This border bottom of pinned columns' header should also be hidden.
**Minimal reproduction of the problem with instructions**

I just used the official plunker [Example - Aligned Grid as Footer](https://www.ag-grid.com/javascript-grid-aligned-grids/#aligned-grid-as-footer), then just replace the theme and add `domLayout: 'autoHeight',` to the footer grid.
**What is the motivation / use case for changing the behavior?**

Make the style consistent.
**Please tell us about your environment:**

win 10
* **ag-Grid version:** 13.3.1

* **Browser:** Chrome, Firefox, Edge

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by makinggoodsoftware** [comment]
Hi,

I have just tried this on my end and I can't reproduce

https://plnkr.co/edit/70EuHjYmTXW4vwRUqBtg?p=preview

**Comment by Teamop** [comment]
yes, I think it has been fixed since v14.2.0, thanks
